package.json

```diff
@@ -3,16 +3,22 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "demo": "nx dev demo",
-    "build": "nx build demo",
-    "start": "nx start demo",
+    "dev": "nx dev next-demo",
+    "build": "nx build next-demo",
+    "start": "nx start next-demo",
+    "remix-demo": "nx dev demo",
```

Comment:
Out of curiosity, are we keeping the remix build? Can we clean up the Remix related scripts?

Reply:
I wanted to keep it around for a little longer.  I think it makes sense to remove the Remix application if/when we start iterating on the demo application.